### PR TITLE
Create gpsbabel-1.5.3.ebuild

### PR DESCRIPTION
**Pull request by P424D0X**
Actual Version 1.5.3 of gpsbabel.

### CODE REVIEW

**Comment by idella** [comment]
despite the fail of the CI test, this pull req appears it could be combined with its counterpart #835, 2 commits both under the one topic in one pull request

**Comment by monsieurp** [comment]
Ditto here.

@gentoo/kde

**Comment by kensington** [comment]
@gentoo/sci

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2014 Gentoo Foundation
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Wrong Copyright date

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/sci-geosciences/gpsbabel/gpsbabel-1.4.4.ebuild,v 1.5 2014/03/02 09:40:59 pacho Exp $
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: This should ```# $Id$```

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+EAPI=5
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Why not ```EAPI=6```?

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+inherit eutils qt4-r2 autotools
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Please sort alphabetically for convenience.

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+DESCRIPTION="GPS waypoints, tracks and routes converter"
+HOMEPAGE="http://www.gpsbabel.org/"
+SRC_URI="
+        http://dev.gentoo.org/~patrick/${P}.tar.gz
+        doc? ( http://www.gpsbabel.org/style3.css -> gpsbabel.org-style3.css )"
+LICENSE="GPL-2"
+
+SLOT="0"
+KEYWORDS="amd64 ~ppc x86 ~x86-fbsd"
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: No stable KEYWORDS on ebuild addition.

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+inherit eutils qt4-r2 autotools
+
+DESCRIPTION="GPS waypoints, tracks and routes converter"
+HOMEPAGE="http://www.gpsbabel.org/"
+SRC_URI="
+        http://dev.gentoo.org/~patrick/${P}.tar.gz
+        doc? ( http://www.gpsbabel.org/style3.css -> gpsbabel.org-style3.css )"
+LICENSE="GPL-2"
+
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Please move this empty line before ```LICENSE```

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+"
+
+PATCHES=(
+        "${FILESDIR}/${PN}-fix_ioapi.patch"
+        "${FILESDIR}/${PN}-1.4.3-fix_binary_name.patch"
+)
+
+src_prepare() {
+        epatch ${PATCHES[@]}
+        epatch_user
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Just use ```EAPI=6```and call ```default``` here.

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+        if use qt4; then
+                dobin gui/objects/gpsbabelfe
+                insinto /usr/share/qt4/translations/
+                doins gui/gpsbabel*_*.qm
+                newicon gui/images/appicon.png ${PN}.png
+                make_desktop_entry gpsbabelfe ${PN} ${PN} "Science;Geoscience"
+        fi
+
+        if use doc; then
+                dohtml gpsbabel.*
```
Comment (jlec) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: ```dohtml```is deprecated. Just use
```ebuild
docinto html
dodoc ${PN}.*
```

File: sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild (hunk 7 above)
Comment (SoapGentoo) [inline sci-geosciences/gpsbabel/gpsbabel-1.5.3.ebuild]: Also, ensure that all `${PATCHES[@]}` are -p1 compliant, as we want to get rid of `epatch`

**Comment by monsieurp** [comment]
Ping @P424D0X?

**Comment by monsieurp** [comment]
Reopen when you'll be willing to apply the suggested changes.

**Pull request closed without merging**